src/Razor/test/Microsoft.CodeAnalysis.Razor.Workspaces.Test/SourceTextExtensionsTest.cs

```diff
@@ -29,7 +29,7 @@ public DateTime GetTheTime()
     {
         return DateTime.Now;
     }
-}");
+}", checksumAlgorithm: SourceHashAlgorithm.Sha256);
```

Comment:
Did this change because the default changed? If so, this should probably call a method on `HashAlgorithmOperations` so that if the default changes again, these tests won't fail.

Reply:
the default seems to be SHA1 and we don't want that

Reply:
`SourceHashAlgorithm` seems to be a `Microsoft.CodeAnalysis.Text` enum and `Microsoft.AspNetCore.Razor.Utilities.Shared` doesn't seem to reference that.

Reply:
Ahh, fair enough then.